### PR TITLE
Publish blog post with the tracing features

### PR DESCRIPTION
**Pull request by fax4ever**

### CODE REVIEW

File: _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+Opening the Jaeger console, we can see that client and server spans are correctly aggregated:
+
+[caption="Jaeger console",link=/assets/images/blog/tracing-hotrod-client.png]
+image::/assets/images/blog/tracing-hotrod-client.png[Tracing output]
+
+You can find a complete application example here: https://github.com/fax4ever/infinispan-play/tree/main/tracing-hotrod-client
+
+== Tracing from a REST client application
+
+The same result can be achieved from a Rest client, putting manually in the HTTP headers requests the current tracing context
```
Comment (dvagnero) [inline _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc:105]: You can achieve the same with a REST client by putting manually in the HTTP headers requests to provide information about the current tracing context using a standard a OpenTelemetry instance of `W3CTraceContextPropagator`. ?

**Review by dvagnero — COMMENTED** [review]
Hi @fax4ever , Great work. I added few comments and suggestions. Please let me know what you think and if I can push these changes. Thanks

File: _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+Basically, if configured, cache events tracing spans will be produced by the Infinispan server and will be delivered to a
+remote tracing collector server (such as Jaeger or Zipkin).
+
+Moreover, if a Java client application with the HotRot or the Rest client produces some tracing spans,
+these spans can be correlated as parent spans of the corresponding spans events produces by the Infinispan server.
+
+== Set up tracing on Infinispan server
+
+The new version of Infinispan server comes with a gRPC OpenTelemetry Protocol (OTLP) exporter,
+and now of most of the tracing servers support it.
```
[the remark this answers is not in the document]
Comment (fax4ever) [reply _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc:25]: +1 sounds good!

File: _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+For instance with the newer Jaeger servers versions it is possible to collect this protocol data enabling the option:
```
[the remark this answers is not in the document]
Comment (fax4ever) [reply _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc:27]: +1 sounds good!

File: _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+[source, shell script]
+----
+./jaeger-all-in-one --collector.otlp.enabled
+----
+
+The server will open a port to import gRPC OTLP tracing data at port 4317.
+
+The tracing on the Infinispan server is configured setting some system properties or environment variables:
```
[the remark this answers is not in the document]
Comment (fax4ever) [reply _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc:36]: ok

File: _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+infinispan.tracing.enabled=true
+otel.service.name=infinispan-server-service
+otel.exporter.otlp.endpoint=http://localhost:4317
+otel.metrics.exporter=none
+----
+
+The first property is Infinispan specific and enables the tracing capability of the Infinispan server.
+The further properties belong to the https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md[OpenTelemetry SDK Autoconfigure project]
+that Infinispan uses to configure the tracing exporter.
+In this case OTLP gRPC exporter protocol is used and the server runs in the same machine of the Infinispan server.
```
[the remark this answers is not in the document]
Comment (fax4ever) [reply _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc:49]: ok

File: _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+(ForkJoinPool.commonPool-worker-2) [org.infinispan.server.core.telemetry.TelemetryServiceFactory] ISPN000952: OpenTelemetry instance loaded: OpenTelemetrySdk{...
+----
+
+To indicate the fact that the OpenTelemetrySdk is correctly configured.
+
+== Tracing from a HotRod client application
+
+Any OpenTelemetry tracing context present on HotRot client applications will be automatically propagated by the new Hot Rod v4 client to the server tracing context.
+
+For instance for a client defining some tracing spans containing cache operations, such as the following:
```
[the remark this answers is not in the document]
Comment (fax4ever) [reply _posts/2022-07-18-infinispan-14-opentelemetry-tracing.adoc:70]: nice!

**Comment by fax4ever** [comment]
Thank you @dvagnero for all the fixes and the suggestions.
All of them look good to me.
Please push all your changes here.

@tristantarrant what do you think? any suggestions? Thanks

**Comment by fax4ever** [comment]
Thanks @tristantarrant

**Comment by fax4ever** [comment]
Merged, thanks @dvagnero!

**Pull request merged**